bin/cobbler-settings

```diff
@@ -114,15 +116,35 @@ def modify(args: argparse.Namespace) -> int:
     schema = migrations.get_schema(migrations.get_installed_version())
     setting_obj = helper.Setting(args.key, args.value)
     # _schema used due to old version of python3-schema in Leap
-    key_type = schema._schema[setting_obj.key_name]
+    try:
+        key_type = schema._schema[setting_obj.key_name]
+    except KeyError:
+        try:
+            key_type = schema._schema[Optional(setting_obj.key_name)]
+        except KeyError:
+            print("Requested key not found in the settings!")
+            return 1
+    if key_type not in (bool, int, float, str, dict, list):
+        if isinstance(key_type, list):
```

Comment:
This is needed because in the schema normally we return something like `list`. However if we have a more detailed schema we may also return something like `[str]`, in those cases we fall into the branch and then we have the two expressions converting the type to one which is of use for us.